images/fedora/f28/Dockerfile

```diff
@@ -13,6 +13,10 @@ COPY README.md /
 RUN sed -i '/tsflags=nodocs/d' /etc/dnf/dnf.conf
 RUN dnf -y swap coreutils-single coreutils-full
 
+COPY missing-docs /
+RUN dnf -y --setopt=tsflags='' reinstall $(<missing-docs)
```

Comment:
It seems like the `--setopt=tsflags=''` isn't necessary. I built an image without it, and found that the packages were reinstalled with their documentation included:
```
host$ buildah bud --tag fedora-toolbox:28 images/fedora/f28
host$ ./toolbox create --release 28
host$ ./toolbox enter --release 28
toolbox$ man gawk
...
<man page shown>
....
```

Reply:
Yes, the flag is not necessary. Fixed.